src/ui/public/state_management/state_storage/hashing_store.js

```diff
@@ -5,15 +5,13 @@ import { Sha256 } from 'ui/crypto';
 import StubBrowserStorage from 'test_utils/stub_browser_storage';
 import { LazyLruStore } from './lazy_lru_store';
 
-const TAG = 'h@';
```

Comment:
I would be OK keeping this here instead of as a static property on the class, but I think a name like `HASH_TAG` makes its role clearer.